### PR TITLE
Make target for dev work flow

### PR DESCRIPTION
**Pull request by mabe13**
Ideas for what make targets would be beneficial for local development. Note currently these targets work but currenlty when running the application the statics aren't being loaded (404s).

### CODE REVIEW

**Comment by stefaniuk** [comment]
Matt, your proposal is good. I refactored the code. Here are the details:
- Moved `AWS_ACCOUNT_ID_*` variables to the `init.sh` file
- Use `pyenv virtualenv`
- Improve `docker-compose-*` make targets
Let me know if you would like to talk about it.
Also, please could you test it?

**Comment by mabe13** [comment]
Note there is an issue with the statics 404ing in when running locally, it appear to be an Django urls issue and needs some other work outside the scope of this task. For this reason it will be noted as a bug and fixed at a later date.

File: Makefile [hunk 1]
```diff
@@ -145,6 +145,11 @@ project-create-ecr:
 dev-setup:
 	make python-virtualenv
 
+dev-clean:
+	make python-virtualenv-clean
+	cd $(APPLICATION_DIR)
+	rm -rfv static/
```
Comment (stefaniuk) [inline Makefile:151]: I wonder if that should be rather a part of the project clean target? If we think we should keep it here as well, can we convert it to `rm -rf $(APPLICATION_DIR)/static`? One line and no `v` flag as the list isn't that important. What do you think Matt?

Comment (mabe13) [reply Makefile:151]: good idea, I'll change it 👍

**Pull request merged**